metrics/lib/features/dashboard/presentation/widgets/sparkline_graph.dart

```diff
@@ -158,10 +158,13 @@ class SparklineGraph extends StatelessWidget {
   DoubleSpan _createSpan(List<num> data) {
     if (data == null || data.isEmpty) return null;
 
-    final sortedData = data.toList()..sort();
+    double min;
+    double max;
```

Comment:
Can we initialize it with first to make sure that we are not doing null check in the loop later?